streaming/README.md

```diff
@@ -511,4 +511,100 @@ metrics, following the same pattern of the receiving side.
 
 For a practical example see [Monitoring ephemeral nodes](#monitoring-ephemeral-nodes).
 
+## Troubleshooting streaming connections
+
+Sometimes when you are configuring master and slave nodes you can have some errors in your log files, in this section it
+is described the most common errors.
+
+### Connection between slave and master is slow
+
+When you have a slow connection between master and slave, Netdata can raise different errors related to this problem.
+The slave Netdata will log a majority of error messages.
+
+```
+netdata ERROR : STREAM_SENDER[SLAVE HOSTNAME] : STREAM SLAVE HOSTNAME [send to MASTER IP:MASTER PORT]: too many data pending - buffer is X bytes long,
+Y unsent - we have sent Z bytes in total, W on this connection. Closing connection to flush the data.
+```
+
+The the value X, Y, Z and W are integer numbers.
+
+On the other hand, in the master side you can have different reports where the most common message that you can see is
+
+```
+netdata ERROR : STREAM_RECEIVER[SLAVE HOSTNAME,[SLAVE IP]:SLAVE PORT] : read failed: end of file
+```
+
+Another common problem is the slave sends only part of the message for the master. In this case, the master
+will not have condition to understand the message received and it will write the following inside `error.log`:
+
+```
+ERROR : STREAM_RECEIVER[SLAVE HOSTNAME,[SLAVE IP]:SLAVE PORT] : sent command 'B' which is not known by netdata, for host 'HOSTNAME'. Disabling it.
+```
+
+In this example, 'B' was part of a 'BEGIN' message that was cut due connection problems.
+
+The last problem related to slow connections happens when some message was missed and the following commands received had a
+direct connection with it. For example, when the slave could not send the list of charts available on its host and it
+sends in the sequence a 'SET' message on a chart that was missed, the master will show a messages like this:
+
+```
+ERROR : STREAM_RECEIVER[SLAVE HOSTNAME,[SLAVE IP]:SLAVE PORT] : requested a SET on chart 'CHART NAME' of host 'HOSTNAME', without a dimension. Disabling it.
+```
+
+### Slave cannot connect to master
+
+Sometimes you are type writing the address and port of the master inside stream.conf and you write a wrong information,
```

Comment:
When the slave can't connect to a master for any reason (misconfiguration, networking, firewalls, master down), you will see the following in the slave's `error.log`.